src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/PkgImportControlTest.java

```diff
@@ -69,6 +71,12 @@ public void setUp() {
         icRootRegexpParent.addChild(icBootRegexpParen);
     }
 
+    @Test
+    public void testDotMetaCharacter() {
+        assertNull(icUncommon.locateFinest("comxkazgroup.courtlink.uncommon.regexp", "MyClass"),
```

Comment:
Can `comxkazgroup` be replaced with something like `com1group'? To avoid growing the whitelist.

Reply:
done